scripts/test_coverage.sh

```diff
@@ -3,7 +3,7 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd $DIR/..
 
-pip install .
+pip install .[test]
```

Comment:
This uh _could_ cause trouble for users with zsh as their shell. But I think it'll probably be fine. Those who that do use it probably can make workarounds